configuration/concepts/activate_forms.csv

```diff
@@ -1,2 +1,2 @@
-Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:2000
-,,,All Observation Templates,All Observation Templates,,ConvSet,N/A,,,Haiti_Vital Signs Form; History and Examination; Second Vitals; Vaccination History; Lab Samples; Radiology Orders,,,,,,,,
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Fully specified name:fr,Short name:fr,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:2000
+,,,All Observation Templates,All Observation Templates,,All Observation Templates,All Observation Templates,,ConvSet,N/A,,,Echantillons de laboratoire; Ordonnances de radiologie,,,,,,,,
```

Comment:
Fix the uuid for the concept similar to how it is in Haiti

Reply:
Done